client/cli/src/main.rs

```diff
@@ -331,6 +335,9 @@ enum Command {
     /// Wipe the notes data from the DB. Useful for testing backup & restore
     #[clap(hide = true)]
     WipeNotes,
+
+    /// Decode a transaction hex string into its JSON representation
```

Comment:
It's actually not encoded to JSON but debug print, right? We'd need to patch in `Serialize` into the `IInput` and `IOutput` traits for that iirc.